script/run

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Run docker-compose in a container
+#
+# This script will attempt to mirror the host paths by using volumes for the
+# following paths:
+#   * $(pwd)
+#   * $(dirname $COMPOSE_FILE) if it's set
+#   * $HOME if it's set
+#
+# You can add additional volumes (or any docker run options) using
+# the $COMPOSE_OPTIONS environment variable.
+#
+
+
+set -e
+
+VERSION="1.4.0dev"
+# TODO: move this to an official repo
+IMAGE="dnephin/docker-compose:$VERSION"
+
+
+# Setup options for connecting to docker host
+if [ -z "$DOCKER_HOST" ]; then
+    DOCKER_HOST="/var/run/docker.sock"
+fi
+if [ -S "$DOCKER_HOST" ]; then
+    DOCKER_ADDR="-v $DOCKER_HOST:$DOCKER_HOST -e DOCKER_HOST"
+else
+    DOCKER_ADDR="-e DOCKER_HOST"
+fi
+
+
+# Setup volume mounts for compose config and context
+VOLUMES="-v $(pwd):$(pwd)"
```

Comment:
That would probably work as well.  There are times when `$PWD` isn't set (if you fork/exec without a shell for example).

Does this cause a problem?